src/cljs/athens/events.cljs

```diff
@@ -1882,31 +1882,25 @@
         {:fx [[:dispatch [:remote/paste-verbatim uid text start value]]]}))))
 
 
-(defn link-unlinked-reference
-  "Ignores case. If title is `test`:
-  test 1     -> [[test 1]]
-  TEST 10    -> [[test 10]]
-  [[attest]] -> [[at[[test]]`"
-  [string title]
-  (let [ignore-case-title (re-pattern (str "(?i)" title))
-        new-str           (string/replace string ignore-case-title (str "[[" title "]]"))]
-    new-str))
-
-
 (reg-event-fx
   :unlinked-references/link
-  (fn [_ [_ block title]]
-    (let [{:block/keys [string uid]} block
-          new-str (link-unlinked-reference string title)]
-      {:dispatch [:transact [{:db/id [:block/uid uid] :block/string new-str}]]})))
+  (fn [_ [_ {:block/keys [string uid] :as block} title]]
```

Comment:
` :as block` not needed, right?

Reply:
Removed. I set `:as block` because it's used as a parameter for the remote event. But since it's removed, the `string` and `uid` are now used directly as the parameters.